tests/bson-serialize-readconcern.phpt

```diff
@@ -0,0 +1,21 @@
+--TEST--
+MongoDB\Driver\ReadConcern serialization
+--FILE--
+<?php
+$tests = [
+	new \MongoDB\Driver\ReadConcern(),
+	new \MongoDB\Driver\ReadConcern( \MongoDB\Driver\ReadConcern::LOCAL ),
+	new \MongoDB\Driver\ReadConcern( \MongoDB\Driver\ReadConcern::MAJORITY ),
+	new \MongoDB\Driver\ReadConcern( \MongoDB\Driver\ReadConcern::LINEARIZABLE ),
+];
+
+foreach ( $tests as $test )
+{
+	var_dump( MongoDB\BSON\toJson( MongoDB\BSON\fromPHP( [ 'rc' => $test ] ) ) );
```

Comment:
Since you removed the restriction on serializing RC, RP, and WC as root documents, is it worth removing the document wrapping (i.e. "rc" field) here?